.github/workflows/trigger-ui-tests.yml

```
@@ -0,0 +1,30 @@
+
+      - name: Get and Trigger Customer Branch UI Tests
+        run: |
+          # Authenticate with GitHub CLI using the token
+          echo ${{ secrets.GITHUB_TOKEN }} | gh auth login --with-token
+
+          git fetch --prune origin
+          BRANCHES=$(git ls-remote --heads origin | awk -F'/' '{print $3"/"$4}' | grep '^customer/')
+          for branch in $(echo "$BRANCHES" | sed -e 's/[\[\]"]//g' -e 's/,/\n/g'); do
+            echo "Triggering UI tests for branch: $branch"
+            gh workflow run automated-manual-tests.yml --ref $branch
+          done
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)